lstchain/tools/lstchain_create_sampling_interval_coefficient_file.py

```diff
@@ -0,0 +1,248 @@
+        help="Third step: Merge sampling coefficient fits files for both gains."
+    ).tag(config=True)
+
+    progress_bar = traits.Bool(
+        help="show progress bar during processing",
+        default_value=True,
+    ).tag(config=True)
+
+    aliases = {
+        ("i", "input"): "EventSource.input_url",
+        ("o", "output"): "SamplingIntervalCoefficientHDFWriter.output",
+        ("g", "gain"): "SamplingIntervalCoefficientHDFWriter.gain",
+        "input_fits": "SamplingIntervalCoefficientHDFWriter.input_fits",
+        "input_fits_hg": "SamplingIntervalCoefficientHDFWriter.input_fits_hg",
+        "input_fits_lg": "SamplingIntervalCoefficientHDFWriter.input_fits_lg",
+        "pedestal": "LSTR0Corrections.drs4_pedestal_path",        
+        "max-events": "EventSource.max_events",
+        "r1_sample_start": "LSTR0Corrections.r1_sample_start",
+        "r1_sample_end": "LSTR0Corrections.r1_sample_end",
+    }
+
+    classes = [LSTEventSource, LSTR0Corrections, SamplingIntervalCalculate]
+
+    flags = {
+        'count_flag': (
+            {'SamplingIntervalCoefficientHDFWriter': {'count_flag': True}},
+            'First step: count peak on each capacitor',
+        ),
+        'stuck_verify_flag': (
+            {'SamplingIntervalCoefficientHDFWriter': {'stuck_verify_flag': True}},
+            'Second step: merge peak count files and verify with charge resolution',
+        ),
+        'merge_flag': (
+            {'SamplingIntervalCoefficientHDFWriter': {'merge_flag': True}},
+            'Third step: Merge sampling coefficient fits files for both gains',
+        )
+    }
+
+    def setup(self):
+        
+        self.sampling_interval_calculate = SamplingIntervalCalculate()
+
+        if self.count_flag:
+            self.setup_count()
+    
+        if self.stuck_verify_flag:
+            self.setup_stuck_verify()
+
+        if self.merge_flag:
+            self.setup_merge()
+
+
+    def setup_count(self):
+        self.log.debug('First step: count peak on each capacitor')
+        self.eventsource = LSTEventSource(parent = self)
+        self.run_id = self.eventsource.obs_ids[0]
+
+    def setup_stuck_verify(self):
+        self.log.debug('Second step: stuck and verify with charge resolution')
+        self.path_list = [str(self.input_fits)]
+        if self.input_fits.is_dir():
+            self.path_list = sorted(self.input_fits.glob(self.glob))
+        self.eventsource = LSTEventSource(parent=self)
+
+    def setup_merge(self):
+        self.log.debug('Third step: Merge sampling coefficients')
+
+
+
+    def start(self):
+        
+        if self.count_flag:
+            self.start_count()
+
+        if self.stuck_verify_flag:
+            self.start_stuck_verify()
+
+        if self.merge_flag:
+            self.start_merge()
+
+    def start_count(self):
+        self.log.debug('start peak counting')
+
+        for i, event in enumerate(self.eventsource):
+            if event.index.event_id % 500 == 0:
```

Comment:
Use `tqdm` for a proper progress bar instead.

Reply:
ok, I didn't understand `tqdm` before:) Now `tqdm` is used during even loop.

Reply:
Then this line is not needed anymore